app/views/people/_gender_field.html.haml

```diff
@@ -0,0 +1,16 @@
+-#  Copyright (c) 2012-2025, Swiss Badminton. This file is part of
+-#  hitobito_swb and licensed under the Affero General Public License version 3
+-#  or later. See the COPYING file at the top-level directory or at
+-#  https://github.com/hitobito/hitobito_swb.
+
+= field_set_tag do
+  %div{data: { controller: 'form-field-toggle', 'form-field-toggle-hide-on-value': Person::GENDERS}}
+    %div
+      = f.labeled(:gender) do
+        - (Person::GENDERS + ['']).each do |key|
+          = f.inline_radio_button(:gender, key, entry.gender_label(key), true, data: { action: 'input->form-field-toggle#toggle'})
+
+    %div{data: { 'form-field-toggle-target': 'toggle' }, class: entry.gender.blank? ? "" : "hidden"}
+      = f.labeled(:ts_gender) do
+        - (Person::GENDERS).each do |key|
+          = f.inline_radio_button(:ts_gender, key, entry.gender_label(key))
```

Comment:
```suggestion
      = render_core_partial 'gender_field', f: f, genders: Person::GENDERS
```

Und core partial erweitern:
```
- genders = local_assigns.fetch(:genders, Person::GENDERS + [''])

= f.labeled(:gender) do
  - (genders).each do |key|
    = f.inline_radio_button(:gender, key, entry.gender_label(key))
```

Reply:
Hab ich mir auch überlegt, aber ich würde da noch warten, bis nochjemand kommt der das braucht. Ich denke ein eigenes Partial ist schonmal ein Schritt in die richtige Richtung